models/docs/sources/history_operations.md

```diff
@@ -1479,3 +1479,7 @@ The identifier of the first reserve asset in a liquidity pool or similar constru
 {% docs details_reserve_b_asset_id %}
 The identifier of the second reserve asset in a liquidity pool or similar construct. This field complements `details.reserve_a_asset_id` in describing the assets involved in a liquidity pool.
 {% enddocs %}
+
+{% docs details_ledgers_to_expire %}
```

Comment:
nit: ledgers_to_expire is deprecated I'm pretty sure. It could be removed if wanted

Reply:
Will keep it for now. If there are other such depricated fields across the board, we can tackle them together as a separate ticket